.eslintrc.yaml

```diff
@@ -117,18 +124,19 @@ rules:
   }, {
     selector: "ThrowStatement > CallExpression[callee.name=/Error$/]",
     message: "Use new keyword when throwing an Error."
-    }, {
-    selector: "CallExpression[callee.object.name='assert'][callee.property.name='fail'][arguments.length=1]",
```

Comment:
This rule should not be removed. It was in a semver-major change: https://github.com/nodejs/node/pull/12293